Add security section for cache poisoning attack (#65)
moovweb-docs/xdn-docs: guides/caching.md

- Request HTTP method
- Request body

> To ensure that you are resilient to [cache poisoning attacks](security#section_cache_poisoning) you must ensure that every request header that influences the rendering of the content is also part of the custom cache key. 

Reviewer: ```suggestion
To ensure that your site is resilient to [cache poisoning attacks](security#section_cache_poisoning) every request header that influences the rendering of the content must be included in your custom cache key. 
```
